utils/main_test.go

```diff
@@ -37,76 +56,91 @@ func TestContainsString(t *testing.T) {
 }
 
 func TestGetUserKey(t *testing.T) {
+	randKey := mkRandRSAPubKey(t)
+
 	table := []struct {
 		key    string
 		user   string
 		keyVal string
 		haserr bool
 	}{
-		{`usera:ssh-rsa AAAA1234 google-ssh {"userName":"usera@example.com","expireOn":"2095-04-23T12:34:56+0000"}`,
-			"usera", `ssh-rsa AAAA1234 google-ssh {"userName":"usera@example.com","expireOn":"2095-04-23T12:34:56+0000"}`, false},
+		{`usera:ssh-rsa [KEY] google-ssh {"userName":"usera@example.com","expireOn":"2095-04-23T12:34:56+0000"}`,
+			"usera", `ssh-rsa [KEY] google-ssh {"userName":"usera@example.com","expireOn":"2095-04-23T12:34:56+0000"}`, false},
+		{`usera:restrict,pty ssh-rsa [KEY] google-ssh {"userName":"usera@example.com","expireOn":"2095-04-23T12:34:56+0000"}`,
+			"usera", `restrict,pty ssh-rsa [KEY] google-ssh {"userName":"usera@example.com","expireOn":"2095-04-23T12:34:56+0000"}`, false},
 		{"    ", "", "", true},
-		{"ssh-rsa AAAA1234", "", "", true},
-		{":ssh-rsa AAAA1234", "", "", true},
+		{"ssh-rsa [KEY]", "", "", true},
```

Comment:
Can you initialize the test data with the key rather than transforming it close to where you are using it? It will make the test easier to read (please apply everywhere you are following this pattern).

Reply:
Done